zipkin-common/src/main/scala/com/twitter/zipkin/test/FunSuiteWithJUnit.scala

```diff
@@ -0,0 +1,9 @@
+package com.twitter.zipkin.test
+
+import org.scalatest.junit.JUnitRunner
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+
+@RunWith(classOf[JUnitRunner])
+class FunSuiteWithJUnit extends FunSuite {
```

Comment:
That works too. Feel free to get started if you want, I'll probably get around to it this weekend otherwise.